src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -487,6 +487,25 @@ suspend fun blockingApplicationLogic(
                             continue@loop
                         }
 
+                        if (healthInformation.medisinskVurdering?.hovedDiagnose?.diagnosekode != null &&
+                                healthInformation.medisinskVurdering.hovedDiagnose.diagnosekode.v == null) {
+                            log.info("Houveddiagnose diagnosekode V mangler", fields(loggingMeta))
+                            val apprec = fellesformat.toApprec(
+                                    ediLoggId,
+                                    msgId,
+                                    msgHead,
+                                    ApprecStatus.AVVIST,
+                                    "Houveddiagnose diagnosekode mangler i sykmeldingen kontaktdin EPJ-levrandør",
```

Comment:
Noen skriveleifer her, siden denne sendes til legen foreslår jeg å rette feilene, f.eks. slik: 
"Diagnosekode for hoveddiagnose mangler i sykmeldingen. Kontakt din EPJ-leverandør."